### PR TITLE
feat: add rerun example

### PR DESCRIPTION
**Pull request by ruben-arts**

### CODE REVIEW

**Comment by ruben-arts** [comment]
@baszalmstra can you try it on windows.

File: examples/rerun/pixi.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[project]
+name = "rerun"
+version = "0.1.0"
+description = "Example package to run the rerun-sdk"
+authors = ["Ruben Arts <ruben@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "win-64", "osx-64"]
+
+[tasks]
+start = "pixi run python dna_exmaple.py "
```
Comment (baszalmstra) [inline examples/rerun/pixi.toml:10]: Doesnt just 

```suggestion
start = "python dna_exmaple.py"
```

work?

Comment (baszalmstra) [reply examples/rerun/pixi.toml:10]: (It does on my machine :) )

**Review by baszalmstra — COMMENTED** [review]
Works flawlessly on Windows 👍

File: examples/rerun/pixi.toml (hunk 1 above)
Comment (ruben-arts) [reply examples/rerun/pixi.toml:10]: Lolll funny that it still works. Pixi is already in my muscle memory 🤣

File: examples/rerun/dna_exmaple.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import rerun as rr
```
Comment (wolfv) [inline examples/rerun/dna_exmaple.py:1]: Typo in file name (example, not exmaple).

**Comment by tdejager** [comment]
Getting this error on osx arm:

```bash
Traceback (most recent call last):
  File "/Users/tdejager/development/prefix/pixi/examples/rerun/dna_example.py", line 1, in <module>
    import rerun as rr
  File "/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_sdk/rerun/__init__.py", line 9, in <module>
    import rerun_bindings as bindings  # type: ignore[attr-defined]
  File "/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_bindings/__init__.py", line 1, in <module>
    from .rerun_bindings import *
ImportError: dlopen(/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_bindings/rerun_bindings.abi3.so, 0x0002): tried: '/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_bindings/rerun_bindings.abi3.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_bindings/rerun_bindings.abi3.so' (no such file), '/Users/tdejager/development/prefix/pixi/examples/rerun/.pixi/env/lib/python3.10/site-packages/rerun_bindings/rerun_bindings.abi3.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64'))
```

**Pull request merged**